### DISCUSSION TITLE
Import of teslafi data unsuccessful

### DISCUSSION THREAD

**Discussion by taugusti** [teslamate-org/teslamate#1311] [Getting started]
**Describe the bug**
I have followed this guide: https://docs.teslamate.org/docs/import/teslafi
The import never ends (spinning pizza forever) and no data is imported 

**Expected behavior**
import should end and teslafi data be available in grafana

**How to reproduce it (as minimally and precisely as possible)**:
0. context: teslamate on synology. simple setup (local only)
1. exported data from teslafi: I have 15 files total 196 MB.
2. copied to a synlogy user folder and from there to the /volume1/@docker/import folder
3. restart teslafi from docker console
4. go to http://x.x.x.x:4000
5. list of files to import are shown
6. start import
7. spinning pizza on the file names forever (>12 hours)
8. when I grow tired of waiting I check x.x.x:3000 (grafana) and se no new data.

**Relevant entries from the logs**
![image](https://user-images.githubusercontent.com/15052792/98023579-6dc28100-1e07-11eb-824a-f6b2cd3b4e12.png)

**Screenshots**

**Data**

**Environment**

- TeslaMate version: 1.20.1
- Type of installation: Docker
- OS TeslaMate is installed on: synology diskstation
- User OS & Browser: win10 / chrome
- Others:

**Comment by adriankumpf** [comment]
On a Synology, the logs can be retrieved via its Docker UI afaik. Without this information, it is not possible to say what is going on.

**Comment by taugusti** [comment]
> On a Synology, the logs can be retrieved via its Docker UI afaik. Without this information, it is not possible to say what is going on.

Hi @adriankumpf 

Great - I was not aware of that.
The log is here:

[docker_teslamate_1.zip](https://github.com/adriankumpf/teslamate/files/5496734/docker_teslamate_1.zip)

I tried to reduce the number of files to import to only two - but nothing changed

I hope you can help - it is much appreciated!

**Comment by adriankumpf** [comment]
Hi @taugusti, can you try the latest development release (available e.g. as `teslamate/teslamate:edge` and `teslamate/grafana:edge` Docker images)?

The import fails because the directory is not readable (probably due to missing permissions). With the current version, this error is actually displayed, and does not lead to a sudden crash as before.

**Comment by hefenglim** [comment]
> Hi @taugusti, can you try the latest development release (available e.g. as `teslamate/teslamate:edge` and `teslamate/grafana:edge` Docker images)?
> 
> The import fails because the directory is not readable (probably due to missing permissions). With the current version, this error is actually displayed, and does not lead to a sudden crash as before.

i meet problem in some month data import

![Screenshot 2021-01-13 200810](https://user-images.githubusercontent.com/21117851/104450493-290f4f00-55db-11eb-9af0-f24ff00d9560.png)

it just 1 second, not import any thing. 

Sample file: 
[TeslaFi112020.zip](https://github.com/adriankumpf/teslamate/files/5808490/TeslaFi112020.zip)

**Comment by hefenglim** [comment]
> > Hi @taugusti, can you try the latest development release (available e.g. as `teslamate/teslamate:edge` and `teslamate/grafana:edge` Docker images)?
> > The import fails because the directory is not readable (probably due to missing permissions). With the current version, this error is actually displayed, and does not lead to a sudden crash as before.
> 
> i meet problem in some month data import
> 
> ![Screenshot 2021-01-13 200810](https://user-images.githubusercontent.com/21117851/104450493-290f4f00-55db-11eb-9af0-f24ff00d9560.png)
> 
> it just 1 second, not import any thing.
> 
> Sample file:
> [TeslaFi112020.zip](https://github.com/adriankumpf/teslamate/files/5808490/TeslaFi112020.zip)

And additional import error i occurs as below msg, seem value overflow handling issues??


![image](https://user-images.githubusercontent.com/21117851/104451168-23fecf80-55dc-11eb-9a41-dc3bb177e350.png)

**Comment by taugusti** [comment]
Hi,
I would love to test this but I have issues with my synology at the moment (have had a synolygy support running for three weeks now) - so it might be a while before I can test it. 
Sorry - I will get back to you asap.
/th

**Reply by choke55** [reply]
Sorry to steel this - but I have an issue to import the csv-files
They never show up after shutdown teslamate and logon to the teslamate interface.
have installed teslamate from this directory volyme2/@docker
have placed my teslafi csv files here --> /volyme2/@docker/import/

hope anyone could give me a hint of the problem

Thanks
/Choke
